CHANGELOG.md

```diff
@@ -9,9 +9,12 @@
 
 ### Bug Fixes
 
-## v1.4.3.0 [unreleased]
-1.  [#3080](https://github.com/influxdata/chronograf/pull/3080): Add tabular data visualization option with features
-1.  [#3103](https://github.com/influxdata/chronograf/pull/3103): Add ability to clone dashboards
+## v1.4.3.1 [2018-04-02]
+### Bug Fixes
```

Comment:
it looks like this is in the wrong place. the UI improvements below should be for v1.5.0.0